plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Tax.php

```diff
@@ -127,9 +127,11 @@ public function is_complete() {
 	 */
 	public function get_additional_data() {
 		return array(
-			'avalara_activated'         => PluginsHelper::is_plugin_active( 'woocommerce-avatax' ),
-			'tax_jar_activated'         => class_exists( 'WC_Taxjar' ),
-			'woocommerce_tax_countries' => self::get_automated_support_countries(),
+			'avalara_activated'              => PluginsHelper::is_plugin_active( 'woocommerce-avatax' ),
+			'tax_jar_activated'              => class_exists( 'WC_Taxjar' ),
+			'woocommerce_tax_activated'      => PluginsHelper::is_plugin_active( 'woocommerce-tax' ),
+			'woocommerce_shipping_activated' => PluginsHelper::is_plugin_active( 'woocommerce-shipping' ),
```

Comment:
Informs [the "Collect sales tax" task](https://github.com/woocommerce/woocommerce/pull/46929/files#r1629393692), [the "Select your shipping options" task](https://github.com/woocommerce/woocommerce/pull/46929/files#r1630344531), and [the help sidebar](https://github.com/woocommerce/woocommerce/pull/46929/files#r1629385636) whether the WC Shipping plugin and the WC Tax plugin are installed.